pyswarms/backend/handlers.py

```diff
@@ -177,13 +177,14 @@ def reflective(self, position, bounds, **kwargs):
             :nowrap:
 
             \begin{gather*}
-                \text{while } x_{i, t, d} \not\in \left[lb_d,\,ub_d\right] \text{ do the
-                following:}
+                \text{while } x_{i, t, d} \not\in \left[lb_d,\,ub_d\right] \\
```

Comment:
Hmm, I didn't see this. Can you give me visual confirmation if this will render properly once generated?
(Just generate the docs and screenshot what this will look like)